### ISSUE TITLE
Impossible to provide new translation

### ISSUE THREAD

**Issue by agrodet** [Tatoeba/tatowiki#4]
I have no idea of what the real problem if or even if there is a problem but here is a message I received from GrizaLeono

> Bien...
> Allant à la page https://fr.wiki.tatoeba.org/art.../premiers-pas# j'ai remarqué que j'étais déjà actif, parce que au dessus de la page je voyais "se déconnecter".
> J'ai cliqué sur "éditer" et j'ai traduit un bon bout de texte, tout en laissant le texte français - ce qui me permet de corriger plus facilement après.
> Quand je devais donner à boire à mon épouse, j'ai cliqué sur "Sauvegarder et continuer".
> Voulant continuer, on me demande de saluer. J'ai salué avec mes anciennes données.
> En claquant sur "éditer" et "Esperanto", je vois des anciens textes...
> J'espère que le travail, que j'ai déjà fait ne sera pas perdu...

The part when he was asked to enter credentials again is probably due to us restarting the wiki. However, I can't explain why his modifications are neither on the French page or the Esperanto page (because I'm not totally sure from where he translated). Does the "Save and continue" button is a temporary save not written in database? I've tested it and it doesn't seem so. 

Maybe @trang or @jiru may have an idea?

**Comment by trang** [comment]
Note that you can see all the recent changes by clicking on [Recent changes](https://eo.wiki.tatoeba.org/history/recent-changes) in the menu of the wiki.

It seems what happened is:
- You created the Esperanto page and initialized it with the French text.
- GrizaLeono went instead to edit the French page.
- GrizaLeono saved modifications for the French page but afterwards went to check the Esperanto page.

That's probably the source of the confusion?

**Comment by agrodet** [comment]
Yes I have checked the Recent changes, and nothing was there. On both pages.  
What you describe is what happened later, when I tried to facilitate the process for GrizaLeono. So I don't know what happened the first time he tried to contribute. It seems that there is no issue with the "save and continue" function, so you can also choose to close that issue since I cannot describe more precisely what happened.

**Comment by agrodet** [comment]
Now, I may have more information. I tried to add the Turkish translation by myself. I filled the title, chose Turkish and filled the URL to display. I added the content of the article. When I click on the Save button, an error is displayed telling me that "This article has already a translation in that language."

As an additional note, the first time I clicked on "Save", I got a message from my browser preventing the page to be displayed because of a potential security risk. After hitting "Return" and trying again, I got the error described above, and it's like that ever since, whatever I try.

**Comment by jiru** [comment]
I tried to add a new French translation by following the steps @agrodet mentioned and it worked. Maybe we missed something when we added the Turkish language?
